doc/architecture/compute/operator-service-endpoints.md

```diff
@@ -0,0 +1,92 @@
+# Compute Service Endpoints
+
+## Status
+  
+  
+### GET /api/v1/operator/status
+   
+   
+Get all workflows and corresponding stats
+
+Parameters
+```
+        owner: String object containg user ETH address (optional)
```

Comment:
why is everything optional? What happens then if I pass no parameter?

Reply:
Fixed. 
At least one input is required (any of them)